R/loopy-helpers.R

```diff
@@ -638,3 +493,31 @@ reorder_pred_cols <- function(x, y_name) {
       .before = dplyr::everything()
     )
 }
+
+engine_to_parsnip <- function(wflow, grid) {
```

Comment:
We have something like this in parsnip. tune _probably_ has something to do that but I could not find it. I'll add an issue to reassess whether we need it or not.